From bfeb70eb7306e37d546c0907f683a3ddef866fd4 Mon Sep 17 00:00:00 2001
Message-ID: <bfeb70eb7306e37d546c0907f683a3ddef866fd4.1701297273.git.sharlatanus@gmail.com>
In-Reply-To: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
References: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 28 Oct 2023 04:29:33 +0100
Subject: [PATCH v3 27/66] gnu: go-github-com-google-safehtml: Remove input
 labels.

* gnu/packages/golang-web (go-github-com-google-safehtml): Remove labels.

Change-Id: I20f5748496988c9e64fa0783bb51507c6cfa933e
---
 gnu/packages/golang-web.scm | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index ce95d1b467..9aa3fe8233 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -163,7 +163,8 @@ (define-public go-github-com-google-safehtml
     (build-system go-build-system)
     (arguments
      '(#:import-path "github.com/google/safehtml"))
-    (propagated-inputs `(("go-golang-org-x-text" ,go-golang-org-x-text)))
+    (propagated-inputs
+     (list go-golang-org-x-text))
     (home-page "https://github.com/google/safehtml")
     (synopsis "Safe HTML for Go")
     (description
-- 
2.41.0

